docker/hadoop/hdp/scripts/sbin/get_hive_home.sh

```diff
@@ -0,0 +1 @@
+export HIVE_HOME=/usr/hdp/$(ls -1 /usr/hdp | grep -v current)/${HIVE_VERSION}
```

Comment:
Can we set the HIVE_HOME via env in this case as well, so we don't need this file?

Reply:
I wanted to but docker ENV only supports static values so we have to find the true value at runtime